scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,383 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+import o3de.command_utils as command_utils
+import o3de.export_project as exp
+import o3de.manifest as manifest
+
+from o3de import android, android_support
+
+
+import argparse
+import logging
+import pathlib
+import platform
+import sys
+import time
+
+from typing import List
+
+def supply_android_default_global_configuration(api:int = 31,
```

Comment:
I would prefer NOT to do this in the script. These environment variables are from the older instructions for the previous android export documentation, and that will eventually be deprecated. If there are any missing values, then the script should halt the process and let the user know which values need to be set. There is an `android-configure --validate` method to do this.